pysat/instruments/cnofs_vefi.py

```diff
@@ -54,7 +80,7 @@
 # support download routine
 # use the default CDAWeb method
 basic_tag = {'dir': '/pub/data/cnofs/vefi/bfield_1sec',
-             'remote_fname': '{year:4d}/'+fname,
+             'remote_fname': '{year:4d}/' + fname,
              'local_fname': fname}
```

Comment:
This is a question.  Does the directory separator change if you access CDAWeb from a windows machine?